<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>947</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Denera Hummel</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="2">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="gridLayoutWidget">
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="2">
         <widget class="QLineEdit" name="lineComment1"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>comment1: </string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>comment2: </string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Name: </string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="lineName"/>
        </item>
        <item row="3" column="2">
         <widget class="QLineEdit" name="lineComment2"/>
        </item>
        <item row="4" column="0">
         <widget class="QComboBox" name="comboBox"/>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QListView" name="listView">
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>947</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menu_Add">
     <property name="title">
      <string>&amp;Add</string>
     </property>
     <addaction name="separator"/>
    </widget>
    <addaction name="menu_Add"/>
    <addaction name="actionSave_2"/>
    <addaction name="actionDelete"/>
    <addaction name="actionContent"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Fi&amp;nd</string>
    </property>
    <widget class="QMenu" name="menuFind_by_Categories">
     <property name="title">
      <string>Find &amp;by Categories</string>
     </property>
     <addaction name="separator"/>
    </widget>
    <widget class="QMenu" name="menu_Find_by_Name">
     <property name="title">
      <string>&amp;Find by Name</string>
     </property>
     <addaction name="separator"/>
    </widget>
    <widget class="QMenu" name="menuFind_on_Content">
     <property name="title">
      <string>Find &amp;on Content</string>
     </property>
     <addaction name="separator"/>
    </widget>
    <addaction name="menu_Find_by_Name"/>
    <addaction name="menuFind_by_Categories"/>
    <addaction name="menuFind_on_Content"/>
   </widget>
   <widget class="QMenu" name="menuHelp_2">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionStatistic"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
   <addaction name="menuHelp_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset>
     <normaloff>alien.gif</normaloff>alien.gif</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normaloff>:/smiles/rar.gif</normaloff>:/smiles/rar.gif</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_AS">
   <property name="text">
    <string>Save AS...</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset>
     <normaloff>:/smiles/flower.gif</normaloff>:/smiles/flower.gif</iconset>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="actionSave_2">
   <property name="icon">
    <iconset>
     <normaloff>:/smiles/gif.gif</normaloff>:/smiles/gif.gif</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>&amp;Delete</string>
   </property>
  </action>
  <action name="actionContent">
   <property name="text">
    <string>&amp;Content</string>
   </property>
  </action>
  <action name="actionStatistic">
   <property name="text">
    <string>&amp;Statistic</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>listView</tabstop>
  <tabstop>lineName</tabstop>
  <tabstop>lineComment1</tabstop>
  <tabstop>lineComment2</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
